griptape/utils/chat.py

```diff
@@ -25,12 +25,16 @@ class Chat:
     )
 
     def default_output_fn(self, text: str) -> None:
-        if self.structure.config.prompt_driver.stream:
+        from griptape.config import config
+
+        if config.drivers.prompt.stream:
```

Comment:
this could be potentially overriden right? the global config is the source of truth for streaming?

Reply:
Great point, updated the logic to look at the actual Tasks.